extensions/ql-vscode/src/view/results/alert-table-header.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+import { useCallback } from "react";
+import { vscode } from "../vscode-api";
+import {
+  InterpretedResultsSortColumn,
+  InterpretedResultsSortState,
+  SortDirection,
+} from "../../common/interface-types";
+import { nextSortDirection } from "./result-table-utils";
+
+export function AlertTableHeader({
+  sortState,
+}: {
+  sortState?: InterpretedResultsSortState;
+}) {
+  const sortClass = useCallback(
+    (column: InterpretedResultsSortColumn): string => {
+      if (sortState !== undefined && sortState.sortBy === column) {
```

Comment:
I believe this should be equivalent (according to the types where `column` cannot be `undefined`):

```suggestion
      if (sortState?.sortBy === column) {
```